ui/tests/acceptance/secrets/backend/totp/key-test.js

```diff
@@ -0,0 +1,173 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: BUSL-1.1
+ */
+
+import authPage from 'vault/tests/pages/auth';
+import { click, fillIn, currentURL, visit, waitUntil } from '@ember/test-helpers';
+import { module, test } from 'qunit';
+import { setupApplicationTest } from 'ember-qunit';
+import { GENERAL } from 'vault/tests/helpers/general-selectors';
+import { SECRET_ENGINE_SELECTORS as SES } from 'vault/tests/helpers/secret-engine/secret-engine-selectors';
+import { mountBackend } from 'vault/tests/helpers/components/mount-backend-form-helpers';
+import { v4 as uuidv4 } from 'uuid';
+import { create } from 'ember-cli-page-object';
+import fm from 'vault/tests/pages/components/flash-message';
+
+const flashMessage = create(fm);
```

Comment:
using anything from `ember-cli-page-object` is deprecated (I think i mentioned this in an earlier review). instead we want to use `sinon` to check flash messages. Here's an [example](https://github.com/hashicorp/vault/blob/main/ui/tests/integration/components/edit-form-test.js#L43). If this is used in multiple test, the spy can be defined in the `beforeEach` hook.